ui/cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
Do we need this? Looks like its all commented code

Reply:
Cypress doesn't like it when I get rid of it